eo.json

```diff
@@ -1382,6 +1382,20 @@
         "Tutorial.Intro.HelpSite" : "Helpo",
 
         "Object.Mirror.Toggle" : "Loka Baskulo",
+        
+        "Tutorial.Streaming.StageControl" : "Sceneja Regilo",
```

Comment:
"Regilo de scenejo" would be clearer. "Sceneja regilo" could mean a controller in the stage, and not a controller for the stage